Products/ZenUI3/browser/resources/js/zenoss/Reports.js

```diff
@@ -26,6 +26,7 @@ report_panel = Ext.create('Ext.panel.Panel', {
     }, {
         xtype: 'panel',
         ref: 'graph_reports',
+        graphs_added: true,
```

Comment:
it should be in `false` only when a node was clicked and a report start loading.